x/axelarcork/keeper/proposal_handler.go

```diff
@@ -109,6 +120,7 @@ func HandleCommunityPoolSpendProposal(ctx sdk.Context, k Keeper, p types.AxelarC
 		return fmt.Errorf("chain by id %d not found", p.ChainId)
 	}
 
+	// TODO(bolten: is there really no fee necessary or executor to target?
```

Comment:
Need an answer to this

Reply:
Yeah, going to ask the Axelar folks today.

Reply:
We will have to account for a fee but might be best done in a follow-up PR.